napari/utils/info.py

```diff
@@ -152,17 +154,29 @@ def sys_info(as_html=False):
         text += f"  - failed to load screen information {e}"
 
     plugin_manager.discover()
-    plugin_strings = []
+    plugin_strings = {}
     for meta in plugin_manager.list_plugin_metadata():
         plugin_name = meta.get('plugin_name')
         if plugin_name == 'builtins':
             continue
         version = meta.get('version')
         version_string = f": {version}" if version else ""
-        plugin_strings.append(f"  - {plugin_name}{version_string}")
+        plugin_strings[plugin_name] = f"  - {plugin_name}{version_string}"
+
+    npe2_plugin_manager = Npe2PluginManager.instance()
+    for manifest in npe2_plugin_manager.iter_manifests():
+        plugin_name = manifest.name
+        if plugin_name in ("napari", "builtins"):
+            continue
+        if version := manifest.package_version:
+            version_string = f": {version}"
+        else:
+            version_string = ""
```

Comment:
Any reason not to use the 1 liner like above?
`version_string = f": {version}" if version else ""`